app/layouts/ProjectLayout.tsx

```diff
@@ -90,10 +79,7 @@ export function ProjectLayout({ overrideContentPane }: ProjectLayoutProps) {
     <PageContainer>
       <TopBar>
         <SiloSystemPicker value="silo" />
-        <ProjectPicker project={project} />
-        {instance && <InstancePicker />}
-        {vpc && <VpcPicker />}
-        {router && <VpcRouterPicker />}
+        <Breadcrumbs />
```

Comment:
All the `TopBar` calls look nearly the same now, so we can eventually move all this into `TopBar`, maybe give it a prop that's `'system' | 'silo'` prop. Doesn't have to be in this PR.